CATALOG.md

```diff
@@ -0,0 +1,142 @@
+# `systelab-cpp-doc` — A catalog of CSW shared libraries for C++
+
+There are a number of libraries that are shared among the different C++ projects that are developed at Werfen Clinical Software. This document is intended to offer a catalog where these libraries and their respective repositories are gathered.
+
+Each library is listed together with a description, the repository where it can be found and other related components that may be of use (if any).
+
+
+## Catalog
+* [cpp-patterns](#cpp-patterns)
+* [cpp-webserver-adapter](#cpp-webserver-adapter)
+* [cpp-encryption-adapter](#cpp-encryption-adapter)
+* [cpp-db-adapter](#cpp-db-adapter)
+* [cpp-gtest-allure-utilities](#cpp-gtest-allure-utilities)
+* [cpp-hash-utils](#cpp-hash-utils)
+* [cpp-json-adapter](#cpp-json-adapter)
+* [cpp-json-settings](#cpp-json-settings)
+* [cpp-jwt-utils](#cpp-jwt-utils)
+* [cpp-rest-api-core](#cpp-rest-api-core)
+* [cpp-test-utilities](#cpp-test-utilities)
+* [cpp-time-adapter](#cpp-time-adapter)
+* [cpp-trace-api](#cpp-trace-api)
+* [cpp-websockets-adapter](#cpp-websockets-adapter)
```

Comment:
This catalog is great! Great job! :)
I would only suggest a couple of things: 
- Adding the new SFTP adapter library (https://github.com/systelab/cpp-sftp-adapter)
- Create another catalog for Bitbucket/private libraries and add a reference to it into this one